src/EA.Weee.Web/Views/OrganisationRegistration/PreviousRegistration.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿@using EA.Weee.Web.Controllers
+@using EA.Weee.Web.RazorHelpers
+@using MvcNavigationHelpers
+@model EA.Weee.Web.ViewModels.OrganisationRegistration.PreviousRegistrationViewModel
+@{
+    ViewBag.Title = "Have you previously been registered as a small producer?";
+}
+
+
+@section full_main_content{
+
+    @(this.WeeeGds().BackLink(@Url.Action("TonnageType", new {searchTerm = Model.SearchText})))
+
+    @using (Html.BeginForm())
+    {
+        @Html.Gds().ValidationSummary()
+
+        @Html.AntiForgeryToken()
+        <header class="hgroup">
+            <h1 class="govuk-heading-l">
+                @ViewBag.Title
+            </h1>
+        </header>
+
+        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.SelectedValue)">
+            @this.WeeeGds().RadioButtonsFor(
+                m => m.SelectedValue,
+                Model.PossibleValues,
+                "Tonnage Type",
```

Comment:
Should this not be "Previous Registration"

Reply:
This is fixed caused.